hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/RecordStore.java

```diff
@@ -57,6 +57,9 @@
 
     boolean tryPut(Data dataKey, Object value, long ttl);
 
+    /**
+     * Returns {@code true} if key doesn't exist previously, otherwise returns {@code false}.
```

Comment:
Can you add more documentation than just the return value? especially what the purpose is of this method,